generator/server_test.go

```diff
@@ -237,10 +237,11 @@ func TestServer_OperationGroups(t *testing.T) {
 	gen.GenOpts.IncludeHandler = true
 	gen.GenOpts.Sections.OperationGroups = []TemplateOpts{
 		{
-			Name:     "opGroupTest",
-			Source:   "asset:opGroupTest",
-			Target:   "{{ joinFilePath .Target .Name }}",
-			FileName: "{{ (snakize (pascalize .Name)) }}_opgroup_test.gol",
+			Name:       "opGroupTest",
+			Source:     "asset:opGroupTest",
+			Target:     "{{ joinFilePath .Target .Name }}",
+			FileName:   "{{ (snakize (pascalize .Name)) }}_opgroup_test.gol",
+			SkipFormat: true,
```

Comment:
【FYI】
If we want to enable format in this test, we also need following logic in `parseGoOrFragment`. I consider implementing it only for test don't make sense enough.
https://github.com/golang/tools/blob/v0.33.0/internal/imports/imports.go#L220-L246